libs/dh/admin/feature-edit-user-role-modal/src/lib/dh-edit-user-role-modal.component.ts

```diff
@@ -58,13 +58,13 @@ import { DhPermissionsTableComponent } from '@energinet-datahub/dh/admin/ui-perm
   standalone: true,
   styles: [
     `
-      :host {
-        display: block;
+      .tabMasterData {
```

Comment:
suggestion:nitpick: consider using kebab case for CSS classes for consistency.

Reply:
Updated naming.